boards/clue_nrf52840/src/main.rs

```diff
@@ -690,10 +690,9 @@ pub unsafe fn main() {
         nrf52840::aes::AesECB<'static>
     ));
 
-    let pconsole =
+    let _pconsole =
```

Comment:
I don't think we need any of these board changes any more.

Reply:
No we don't, I reverted the changes.